code/modules/assembly/bang.dm

```diff
@@ -0,0 +1,55 @@
+/obj/item/assembly/bang
+	name = "BANG"
+	desc = "A small electronic device able to emit a loud sound."
+	icon_state = "igniter"
```

Comment:
You sure people aren't gonna get confused?

Reply:
>Needs icon(s).